lib/services/http-service.js

```diff
@@ -335,19 +335,41 @@ function httpServiceFactory(
      * @returns
      */
     function createHttpProxy(app, httpProxy){
+        var sep = path.sep; //The platform-specific file separator. '\\' or '/'
         try {
-            var localPath = httpProxy.localPath || '/',
-                remotePath = httpProxy.remotePath || '/',
+            var localPath = httpProxy.localPath || sep,
+                remotePath = httpProxy.remotePath || sep,
                 serverAddr = httpProxy.server;
-            app.use(localPath, proxy(serverAddr, {
-                forwardPath: function (req) {
-                    return path.join(remotePath, require('url').parse(req.url).path);
-                },
-                decorateRequest: function (req) {
-                    //For HTTPS access
-                    req.rejectUnauthorized = false;
-                }
-            }));
+
+            //localPath & remotePath should always starts with /
+            if (!_.startsWith(localPath, sep)) {
```

Comment:
FYI We also support the ES6 native `String.startsWith` method because of the es6-shim library within di.js (https://github.com/RackHD/di.js/blob/master/package.json#L16)